PLEP-0007.rst

```diff
@@ -0,0 +1,262 @@
+|   A sub-package that fully defines a plasma.  This would include the         |
+|   plasma's species constituents and physical parameters (like temperature,   |
+|   boundary conditions, magnetic fields, etc.).                               |
+|                                                                              |
+|   Any tools that go into defining a plasma or its environment (e.g. a field  |
+|   solver) should be included in a sub-package within ``plasmapy.plasma``.    |
++------------------------------------------------------------------------------+
+| .. _simulation:                                                              |
+|                                                                              |
+| **simulation**                                                               |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/simulation/``                                       |
+| :access: ``plasmapy.simulation``                                             |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A sub-package focused on interfacing with simulations and/or running       |
+|   simulations.                                                               |
+|                                                                              |
+|   If a new feature falls under the scope of the ``analysis`` and/or          |
+|   ``diagnostics`` sub-packages, then the feature should be included one of   |
+|   respective sub-packages.  For example, a synthetic diagnostic should be    |
+|   included in the ``plasmapy.diagnostics`` sub-package.                      |
++------------------------------------------------------------------------------+
+| .. _tests:                                                                   |
+|                                                                              |
+| **tests**                                                                    |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/tests/``                                            |
+| :access: ``plasmapy.tests``                                                  |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of tests for top-level modules (i.e. functions and classes    |
+|   defined in top-level ``.py`` files).                                       |
+|                                                                              |
+|   Utility's associated with running and developing tests (e.g.               |
+|   ``pytest_helpers``) should also be included here over ``plasmapy.utils``.  |
```

Comment:
I'd remove the ```(e.g., ``pytest_helpers``)``` since the new place I'm putting things is `plasmapy.tests.helper` (which is akin to `astropy.tests.helper` to have some consistency between packages).

Reply:
(Also, for consistency with `plasmapy.utils`, should this be a `.. note::` thing too?

Reply:
To make things less specific I quoted "pytest helpers" instead of directly naming ``pytest_helpers``.   I like leaving the example in there and the new wording does not restrict us to a sub-package structure.

I also made this a ``.. note::``